nextjs-task-system/pages/api/comments/index.ts

```
@@ -0,0 +1,92 @@
+  };
+
+  if (method === "GET") {
+    // Solo los usuarios con rol ADMIN pueden leer comentarios
+    if (user.role !== "ADMIN") {
+      return res.status(403).json({ error: "Access denied. Only admins can view comments." });
+    }
+
+    try {
+      const taskId = query.taskId ? parseInt(query.taskId as string, 10) : null;
+
+      if (!taskId || isNaN(taskId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)